### PR TITLE
Bugfix FXIOS-4561 [v106] Reload button is not functional after focusing the URL bar #11298

### PR DESCRIPTION
**Pull request by mozead1996**
# [FXIOS-4561](https://mozilla-hub.atlassian.net/browse/FXIOS-4561) https://github.com/mozilla-mobile/firefox-ios/issues/11298
Reload button is not functional after focusing the URL bar

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -901,7 +901,6 @@ class BrowserViewController: UIViewController {
             }
         })
         view.setNeedsUpdateConstraints()
-        urlBar.locationView.reloadButton.reloadButtonState = .disabled
```
Comment (mozead1996) [inline Client/Frontend/Browser/BrowserViewController.swift:904]: this line here set the reloadButtonStatus to **disabled**  and it's called each time when focusing out of the URL bar

**Comment by jevans-mozilla** [comment]
Thanks for this fix.  The URL bar has a lot of interactions with other parts of the code so we are looking to add some tests around this fix to ensure we aren't impacting another area.

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 2]
```diff
@@ -894,7 +894,7 @@ class BrowserViewController: UIViewController {
             self.webViewContainer.accessibilityElementsHidden = true
             UIAccessibility.post(notification: UIAccessibility.Notification.screenChanged, argument: nil)
         })
-        urlBar.locationView.reloadButton.reloadButtonState = .disabled
+        view.setNeedsUpdateConstraints()
```
Comment (lmarceau) [inline Client/Frontend/Browser/BrowserViewController.swift:897]: `view.setNeedsUpdateConstraints()` was moved to line 878. This should be removed.

**Comment by lmarceau** [comment]
@mozead1996 something went wrong, this was a one line change and there's now multiple line changes.
Screen Shot 2022-08-10 at 4 11 55 PM

**Comment by mozead1996** [comment]
@lmarceau  i updated the file but i see other changes not sure why , i just pulled from the upstream not the fork

**Comment by lmarceau** [comment]
> @lmarceau i updated the file but i see other changes not sure why , i just pulled from the upstream not the fork

Indeed, the rebase didn't work. Maybe best is to create a new PR? Should be quick as this is a one line change.

**Pull request closed without merging**